.github/ISSUE_TEMPLATE/feature_request.ja.yml

```diff
@@ -1,33 +1,33 @@
-name: "Feature Request (日本語)"
-description: "新しい機能のリクエストや改善を提案します"
-labels: ["enhancement"]
+name: "Component Request (日本語)"
+description: "新しいコンポーネントのリクエストや改善を提案します"
+labels: ["triage", "enhancement"]
 body:
   - type: "markdown"
     attributes:
       value: |
-        機能のリクエストを作成していただきありがとうございます！ 😎
+        コンポーネントのリクエストを作成していただきありがとうございます！ 😎
 
-        新しい機能のトピックについてのアイデアがある、何かが適切に機能されていないと気づいた、または現在の機能に何か誤りがあると感じた場合は、ここが正解です！
+        新しいコンポーネントについてのアイデアがある場合は、ここが正解です！
   - type: "input"
     id: "subject"
     attributes:
       label: "主題"
-      description: "この機能の主題は何ですか？"
-      placeholder: "コントラスト"
+      description: "このコンポーネントの主題は何ですか？"
+      placeholder: "Header"
```

Comment:
Is it okay to leave this placeholder in English?

Reply:
No problem.Component names are always in English.